NorthstarLauncher/main.cpp

```diff
@@ -324,6 +324,14 @@ HMODULE LoadDediStub(const char* name)
 int main(int argc, char* argv[])
 {
 
+	if (strstr(GetCommandLineA(), "-waitfordebugger"))
+	{
+		while (!IsDebuggerPresent())
+		{
+			Sleep(100);
```

Comment:
```suggestion
			// Sleep 100ms to give debugger time to attach.
			Sleep(100);
```

Reply:
![image](https://user-images.githubusercontent.com/66967891/200684510-a123c0d5-3429-49bb-894c-860f0c7efe18.png)
yes, github, that is exactly what they did to that code.

In all seriousness, commenting this I would say is pretty important